src/screenshot.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { describe } from "jest-circus";
+// @ts-ignore needed for the mock
+import puppeteer from "puppeteer";
```

Comment:
Why did you add those two lines ? I tried to run the tests without them and everything seems fine

Reply:
At some point when I launched the tests without this line one test was failing during the run but not at the end. So it makes me less confident but I can't reproduce it either explain why it did this 🤔

Reply:
I removed it 👌